source/error.c

```diff
@@ -150,9 +150,8 @@ void aws_register_error_info(const struct aws_error_info_list *error_info) {
     AWS_ASSERT(error_info->error_list);
     AWS_ASSERT(error_info->count);
 
-    int min_range = error_info->error_list[0].error_code;
-
-    int slot_index = min_range >> SLOT_DIV_SHIFT;
+    const int min_range = error_info->error_list[0].error_code;
+    const int slot_index = min_range >> SLOT_DIV_SHIFT;
```

Comment:
Any reasonable compiler will do strength reduction if you do division by a constant power of 2. Would be easier to read.

Reply:
I'm not sure I understand, what would be your proposed implementation?